assets/models/system/deformable_detr_twostage_refine_r50_16x2_50e_coco/description.md

```diff
@@ -1,9 +1,12 @@
+
 `deformable_detr_twostage_refine_r50_16x2_50e_coco` model is from <a href="https://github.com/open-mmlab/mmdetection/tree/v2.28.2" target="_blank">OpenMMLab's MMDetection library</a>. This model is reported to obtain <a href="https://github.com/open-mmlab/mmdetection/blob/e9cae2d0787cd5c2fc6165a6061f92fa09e48fb1/configs/deformable_detr/metafile.yml#L55" target="_blank">box AP of 46.8 for object-detection task on COCO dataset</a>. To understand the naming style used, please refer to <a href="https://mmdetection.readthedocs.io/en/v2.28.2/tutorials/config.html#config-name-style" target="_blank">MMDetection's Config Name Style</a>.
 
 DETR has been recently proposed to eliminate the need for many hand-designed components in object detection while demonstrating good performance. However, it suffers from slow convergence and limited feature spatial resolution, due to the limitation of Transformer attention modules in processing image feature maps. To mitigate these issues, we proposed Deformable DETR, whose attention modules only attend to a small set of key sampling points around a reference. Deformable DETR can achieve better performance than DETR (especially on small objects) with 10 times less training epochs. Extensive experiments on the COCO benchmark demonstrate the effectiveness of our approach.
 
 > The above abstract is from MMDetection website. Review the <a href="https://github.com/open-mmlab/mmdetection/tree/v2.28.2/configs/deformable_detr" target="_blank">original-model-card</a> to understand the data used to train the model, evaluation metrics, license, intended uses, limitations and bias before using the model.
 
+<b>Deprecation Warning</b>: This model version is only compatible with mmdet <=2.28 and is deprecated. We recommend to use the corresponding model compatible with mmdet >= 3.1.0. In our model catalog, the models prefixed with mmdet-3x are compatible with mmdet >= 3.1.0.
```

Comment:
Better to specify new name to look in the catalog

Reply:
Great. Added the name of the 3.x model. The link should work once we release mmd-3x models to azureml.